dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -1810,6 +1808,10 @@ private ResourceInfo updateResourceInfo(ResourceInfo res) {
             resourceInfo = new ResourceInfo();
             // get resource from database, only one resource should be returned
             Resource resource = getResourceById(resourceId);
+            if (Objects.isNull(resource)) {
```

Comment:
Hi, @LiJie20190102 , May I ask is there any other place that needs to check if the resource is null like this?